static/js/components/ThumbnailList.jsx

```diff
@@ -10,7 +10,9 @@ import styles from "./ThumbnailList.css";
 dayjs.extend(calendar);
 
 
-const Thumbnail = ({ ra, dec, telescope, observed_at, name, url }) => {
+const Thumbnail = ({ ra, dec, telescope, mjd, name, url }) => {
+  const unixt = (mjd - 40587.0) * 86400000;
```

Comment:
Add a small comment to document where 40587.0 comes from ... assuming it's an mjd to unix time conversion? What does the whole equation give you?

Reply:
The whole equation converts the given mjd into a unix timestap in ms

Reply:
40587 is the MJD of the last UNIX timestamp epoch (1970-01-01)